Zigbee FAQ about missing links in ZHA visualization (#33419)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the message propagates through the mesh networking matrix.

#### Why some links are missing in Zigbee network topology maps

> Missing links between Zigbee End Devices (i.e., battery-powered devices) in the Zigbee network visualization map are usually nothing to worry about. This is a common occurrence and is generally not a problem or fault if the device is still reporting state changes. This happens with sleepy Zigbee End Devices and does not mean that the device is no longer connected.

Reviewer: ```suggestion
Missing links between Zigbee end devices (often battery-powered devices) in the Zigbee network visualization map are common. They are generally not a sign of a faulty device if the device is still reporting state changes. This happens with sleepy Zigbee end devices and does not mean that the device is no longer connected.
```

> The reason for this is that some end devices (e.g., Xiaomi door sensors) sleep for an extended period, causing the parent Zigbee Router to remove them from its child table via a feature called router child ageing. Since using child ageing and removing it from the child table is a Zigbee 3.0 feature, this does not always occur because not all Zigbee Router devices use child ageing.

Reviewer: ```suggestion
Some end devices (for example, Xiaomi door sensors) sleep for an extended period, causing the parent Zigbee Router to remove them from its child table via a feature called router child aging. Since using child aging and removing them from the child table is a Zigbee 3.0 feature, this does not always occur because not all Zigbee router devices use child aging.
```

> This is what causes devices to show with a missing link. Even though the device is not in the child table anymore, the end device can still communicate via the parent Zigbee Router.

Reviewer: ```suggestion
This is what causes devices to show a missing link. Even though the device is no longer in the child table, the end device can still communicate via the parent Zigbee router.
```
